challenge_templates/xss/src/main.py

```diff
@@ -0,0 +1,63 @@
+import os
+import secrets
+import asyncio
+
+from urllib.parse import parse_qs
```

Comment:
pep8 groupings. Put stdlib in one grouping at the top (even if `from...`), then 3rd party, then local libs